js/app/controllers/calcontroller.js

```diff
@@ -140,15 +140,20 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 				$scope.$apply();
 			});
 		} else {
-			$scope.calendarsPromise = CalendarService.getPublicCalendar(constants.publicSharingToken).then(function(calendar) {
+			var splitted = constants.publicSharingToken.split(".")
+			var i;
+			for(i = 0; i < splitted.length; i++ )
```

Comment:
You could also use `constants.publicSharingToken.split(".").forEach(token => {...}` here :)